dev-packages/e2e-tests/test-applications/sveltekit-2-svelte-5/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "sveltekit-2-svelte-5",
+    "version": "0.0.1",
+    "private": true,
+    "scripts": {
+        "dev": "vite dev",
+        "build": "vite build",
+        "preview": "vite preview",
+        "proxy": "node start-event-proxy.cjs",
+        "clean": "npx rimraf node_modules,pnpm-lock.yaml",
+        "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+        "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+        "test:prod": "TEST_ENV=production playwright test",
+        "test:build": "pnpm install && npx playwright install && pnpm build",
+        "test:assert": "pnpm test:prod"
+    },
+    "dependencies": {
+        "@sentry/sveltekit": "latest || *",
+        "@spotlightjs/spotlight": "2.0.0-alpha.1"
```

Comment:
just to clarify, to we want to leave this in here?

Reply:
yeah I think it doesn't hurt and it helped with local inspection of envelope payloads.